AdAway/src/main/res/values-de/strings_preferences.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_enable_systemless_def">falsch</string>
```

Comment:
Has anybody something to say about this?
As I already said: I am not sure this is not breaking thinks, because "falsch" has to be parsed properly to be boolean.

Reply:
And "richtig" (which should be "wahr") will be parsed as " false"... Everything except "true" will be false.